README.md

```diff
@@ -916,6 +917,29 @@ module "setup_tfstate" {
 | s3_bucket_arn  | string | The ARN of the s3 bucket  |
 | s3_bucket_name | string | The name of the s3 bucket |
 
+# Outbound proxy VPC
+
+Whenever you need to talk to APIs which use IP based whitelisting, this is
+the module to create the required setup with. It requires an eip/elastic ip
+and it spits out a vpc_config which can be attached to a lambda function. The
+lambda function will then execute requests via the ip of the given eip. For an
+example, click [here](./examples/aws/lambda_with_outbound_proxy/main.tf).
```

Comment:
can we add a code block where all options are shown/used, similar to other modules?